src/auth/middleware.ts

```diff
@@ -1,27 +1,41 @@
-import muid from 'uuid-mongodb'
+import muid, { MUUID } from 'uuid-mongodb'
 import { AuthUserType } from '../types.js'
 import { verifyJWT } from './util.js'
 
 /**
  * Create a middleware context for Apollo server
  */
-export const createContext = async ({ req }): Promise<any> => {
-  const { headers } = req
+export const createContext = (() => {
+  let testUUID: MUUID
 
-  const user: AuthUserType = {
-    roles: [],
-    uuid: undefined
+  if (process.env.NODE_ENV === 'development') {
+    testUUID = muid.v4()
+    console.log(`The user.uuid for this session is: ${testUUID.toString()}`)
   }
 
-  const authHeader = String(headers?.authorization ?? '')
-  if (authHeader.startsWith('Bearer ')) {
-    const token = authHeader.substring(7, authHeader.length).trim()
-    const z = await verifyJWT(token)
+  return async ({ req }): Promise<any> => {
+    const { headers } = req
 
-    user.roles = z?.['https://tacos.openbeta.io/roles'] ?? []
-    const uidStr: string | undefined = z?.['https://tacos.openbeta.io/uuid']
-    user.uuid = uidStr != null ? muid.from(uidStr) : undefined
-  }
+    const user: AuthUserType = {
+      roles: [],
+      uuid: undefined
+    }
+
+    if (process.env.NODE_ENV === 'development' && (user.uuid == null)) {
```

Comment:
What do you think if we require the user to explicitly opt-in instead of relying on the default `NODE_ENV` var?  Maybe create a new variable "GOD_MODE=true" or LIZARD_MODE or whatever you want to call it, and set it before calling `dev-serve` in package.json. 

My concern is in production we might inadvertently unset `NODE_ENV=production` or forget to set it when working with the docker file or devops automation task.

Reply:
I think this is a good idea. I became concerned about this myself when I noticed that while the tests are running, NODE_ENV is set to 'test'. I'll change it to GOD_MODE=true.

I am not sure what you mean by "before calling `dev-serve`". I'm going to stick with the current strategy, where `serve-dev` sets the env variable, but if you want me to do something else, let me know and I'll change it.

Reply:
> I am not sure what you mean by "before calling dev-serve". I'm going to stick with the current strategy, where serve-dev sets the env variable, but if you want me to do something else, let me know and I'll change it.

Sorry I mixed up serve-dev and dev-serve.  This is what I meant:

package.json
```json
"serve-dev": "GOD_MODE=true yarn serve"
```